draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -654,476 +526,437 @@
+   	all trigger resource representations for a specific trigger state (e.g. all triggers in "pending" state).
    </t>   
    <t>
-      uCDN SHOULD request the extended Trigger Collection view by passing the query string parameter 
-      "status=extended" when requesting a Trigger Status Resource Collection.
-      dCDN SHOULD appropriately respond to the request for extended status with HTTP Status 200 ("OK") 
-      when such request can be satisfied, with error code 501 ("Not Implemented") if the capability 
-      has not been implemented or advertised, and 400 ("Bad Request") in case of an unrecognized query parameters.
+      	uCDN SHOULD request the extended representation by passing the query string parameter 
+   	"status=extended" when requesting a trigger collection resource or one of its filtered representations.
+      	dCDN SHOULD appropriately respond to the request for extended status with HTTP Status 200 ("OK") 
+      	when such request can be satisfied, with error code 501 ("Not Implemented") if the capability 
+      	has not been implemented or advertised, and 400 ("Bad Request") when dCDN encounters a malformed query format.
+   </t>
+   <t>
+	By default, trigger resources are represented in the trigger collection with their resource URL only.
    </t>
+
    </section>
    </section>
 
-   <section title="Polling Trigger Status Resources" anchor="polling-trigger-resources"><t>
-   uCDN has a URI provided by dCDN for each Trigger Status
-   Resource it has created. It may fetch that Trigger Status Resource
-   at any time.</t>
-
-   <t>
-   This can be used to retrieve progress information and to fetch the
-   result of the CI/T Command.</t>
-
+   <section title="Polling Triggers" anchor="polling-trigger-resources">
    <t>
-   When polling in this way, uCDN SHOULD use HTTP ETags to monitor
-   for change, rather than repeatedly fetching the Trigger Status
-   Resource.</t>
-
+	uCDN has a URI provided by dCDN at the trigger creation time. 
+	Alternatively, uCDN MAY discover trigger resource URI by retrieving the trigger collection resource and/or one of its filtered 
+	representations.
+	It may fetch an up-to-date representation of the trigger resource at any time using HTTP GET request, including changes in trigger
+	state, as well as the outcome of the trigger processing.
+   </t>
    </section>
 
    </section>
 
-   <section title="Cancelling Triggers" anchor="cancelling-triggers">
+   <section title="Deleting Triggers" anchor="deleting-triggers">
    <t>
-   uCDN can request cancellation of a trigger by issuing CI/T Modify 
-   Trigger Command to the specific trigger's Trigger Status Resource URL 
-   and setting the "desired-status" field to "cancelled".
+   	uCDN MAY request delete trigger resources at any time using the HTTP DELETE method. 
+	As defined in the CDNI Control Interface / Triggers 1st edition <xref target="RFC8007"/>,
```

Comment:
Punctuation and grammar issue:  "uCDN may request the deletion of trigger resources at any time using the HTTP DELETE method, as defined in the CDNI Control Interface / Triggers 1st edition <xref target="RFC8007"/>."

Reply:
fixed